.github/workflows/ci.yml

```diff
@@ -79,6 +103,29 @@ jobs:
       - run: npm ci
       - run: npm run lint:frontend
 
+  backend_test:
+    needs: [changes, install, build]
+    # if: ${{ needs.changes.outputs.backend == 'true' }}
```

Comment:
should we reenable this check too? it should only run if there are backend changes imo

Reply:
yup will reenable after it's approved! wanted to show that the separate flows run successfully

Reply:
[af93bca](https://github.com/opengovsg/FormSG/pull/5483/commits/af93bcac96b49e9550eea3074776d4f226fdfdbf)